app/javascript/mastodon/locales/en.json

```diff
@@ -525,14 +523,14 @@
   "poll_button.add_poll": "Add a poll",
   "poll_button.remove_poll": "Remove poll",
   "privacy.change": "Change post privacy",
-  "privacy.direct.long": "Visible for mentioned users only",
-  "privacy.direct.short": "Mentioned people only",
-  "privacy.private.long": "Visible for followers only",
-  "privacy.private.short": "Followers only",
-  "privacy.public.long": "Visible for all",
+  "privacy.direct.long": "Everyone mentioned in the post",
+  "privacy.direct.short": "Specific people",
+  "privacy.private.long": "Only your followers",
+  "privacy.private.short": "Followers",
+  "privacy.public.long": "Anyone on and off Mastodon",
   "privacy.public.short": "Public",
-  "privacy.unlisted.long": "Visible for all, but opted-out of discovery features",
-  "privacy.unlisted.short": "Unlisted",
+  "privacy.unlisted.long": "Public with less reach",
+  "privacy.unlisted.short": "Low-key public",
```

Comment:
I am not a fan of changing existing strings, and in this case, the new strings do not seem clearer.

Arguably, all options but the most public ones are “Specific people” only, “Low-key public” is not more understandable than “Unlisted” (although I understand “Unlisted” is often misunderstood as well), “Public with less reach” is not more understandable than “Visible for all, but opted-out of discovery features”.

Also you are describing a group of people, but not what these people can do. It could conceivably be confused with e.g. reply controls.

Reply:
This can be iterated on after the holidays, when Sam is back.

Reply:
I guess, but this will be all the more iterations translators will have to work with :/